website/docs/docs/deploy/job-scheduler.md

```diff
@@ -67,32 +75,82 @@ Jobs consume a lot of memory in the following situations:
 Refer to [dbt Cloud architecture](/docs/cloud/about-cloud/architecture) for an architecture diagram and to learn how the data flows.
 
 
-<!-- leaving space for CI job treatment, runtime, warm up pods updates
+### Pod memory 
 
--->
 
-## Run cancelation for over-scheduled jobs
+## Run cancellation for over-scheduled jobs
 
 :::info Scheduler won't cancel API-triggered jobs 
 The scheduler will not cancel over-scheduled jobs triggered by the [API](/docs/dbt-cloud-apis/overview).
 :::
 
-**Over scheduled jobs** &mdash; The dbt Cloud scheduler prevents too many job runs from clogging the queue by canceling unnecessary ones. If a job takes longer to run than its scheduled frequency, the queue will grow faster than the scheduler can process the runs, leading to an ever-expanding queue with runs that don’t need to be processed. 
+The dbt Cloud scheduler prevents too many job runs from clogging the queue by canceling unnecessary ones. If a job takes longer to run than its scheduled frequency, the queue will grow faster than the scheduler can process the runs, leading to an ever-expanding queue with runs that don’t need to be processed (called _over-scheduled jobs_). 
 
 The scheduler prevents queue clog by canceling runs that aren't needed, ensuring there is only one run of the job in the queue at any given time. If a newer run is queued, any previous queued run for that job will be canceled and have a helpful error message displayed:
 
 <Lightbox src="/img/docs/dbt-cloud/deployment/run-error-message.jpg" width="85%" title="The cancelled runs display a helpful error message explaining why the run was cancelled and recommendations"/>
 
 To prevent over-scheduling, users will need to take action by either refactoring the job so it runs faster or modifying its [schedule](/docs/deploy/job-triggers).
 
+## Create and schedule jobs {#create-and-schedule-jobs}
```

Comment:
Can we move this into it's own page? I don't think it belongs in the scheduler page. You can create a new page on the side bar called "Job settings"